Mathlib/RingTheory/Algebraic/Pi.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+import Mathlib.Algebra.Polynomial.AlgebraMap
+
+/-!
+# Algebraic functions
+
+This file defines algebraic functions as the image of the `algebraMap R[X] (R → S)`.
+-/
+
+assert_not_exists IsIntegralClosure
+assert_not_exists LinearIndependent
+assert_not_exists LocalRing
+assert_not_exists MvPolynomial
+
+universe u v w
+
+open Polynomial
+
+section Pi
+
+variable (R' : Type u) (S' : Type v) (T' : Type w)
```

Comment:
Maybe `variable (R S T : Type*)` is OK in this file, since we don't use R, S, T in the other places of this file, and we don't use `Cardinal.lift`, etc. which requires explicit universe levels.